cirq-google/cirq_google/engine/engine_client.py

```diff
@@ -124,7 +130,7 @@ def __init__(
 
     @cached_property
```

Comment:
I'd suggest removing this `cached_property` decorator since there's no need to cache on each `EngineClient` instance as well as on the `AsyncExecutor` class.

Reply:
Done